<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetsViewerWidget</class>
 <widget class="QWidget" name="AssetsViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assets</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="3">
      <widget class="QGroupBox" name="inputsGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>180</height>
        </size>
       </property>
       <property name="title">
        <string>Inputs</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="valueLabel">
            <property name="text">
             <string>Value</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>valueDoubleSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="valueDoubleSpinBox"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Duration in years</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>durationDoubleSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="durationDoubleSpinBox"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="modeLabel">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QComboBox" name="modeComboBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="dateLabel">
            <property name="text">
             <string>Date</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>dateEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="displayFormat">
             <string notr="true">dd/MM/yy</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="detailsLabel">
            <property name="text">
             <string>Details</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="commentsLabel">
            <property name="text">
             <string>Comments</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="commentsEdit">
            <property name="minimumSize">
             <size>
              <width>183</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="detailsEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="bankComboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="bankLabel">
            <property name="text">
             <string>Bank</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="yearLabel">
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDateEdit" name="yearEdit">
            <property name="displayFormat">
             <string notr="true">dd/MM/yy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="declareLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Value to declare</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="recordButton">
       <property name="text">
        <string>&amp;Record</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>valueDoubleSpinBox</tabstop>
  <tabstop>durationDoubleSpinBox</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>detailsEdit</tabstop>
  <tabstop>commentsEdit</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
